src/pages/api/checkout_sessions/checkout_session.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import Stripe from "stripe";
+
+const stripe = new Stripe("pk_test_L1f0e3XAzjsG7jtp4uN7L9ql", {
```

Comment:
Could you make this API key an environment variable that can be configured in .env?

Reply:
oh whoops forgot I put in a test api to test but it didn't work since they don't accept published apikeys.